### PR TITLE
Feature/lz4 codec

### PR DESCRIPTION
**Pull request by zerosign**
no test yet

### CODE REVIEW

File: lib/kafka/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module Kafka
-  VERSION = "0.5.1"
+  VERSION = "0.5.2"
```
Comment (dasch) [inline lib/kafka/version.rb]: There's no reason to bump the ruby-kafka version.

**Comment by dasch** [comment]
Can you add a unit test of Kafka:: LZ4Codec?

**Comment by dasch** [comment]
Also, thanks for this (:heart:) and sorry for the late response.

**Comment by zerosign** [comment]
sorry for the late response too ..

**Pull request merged**